templates/grouping.html

```diff
@@ -0,0 +1,43 @@
+<label for="grouping">
+  <div>Repeat rows?
+    <span aria-label="(Required field)"></span>
+  </div>
+</label>
+<ul id="qb-grouping-list">
+  <li>
+    <label>
+      <input type="radio" name="grouping" value="summary" id="grouping-1" aria-label="Summary"
+      {% if not grouping or (grouping and grouping == "summary") %}
+        checked="checked"
+      {% endif %}
+      />
+      <span class="tooltip-ui" data-toggle="tooltip" title="Information is not disaggregated, leading to repeated rows.">
+        No
+      </a>
```

Comment:
There are no opening `<a>` tag, though several closing ones.

Looks like they should be `</span>` instead.